include/pokemon.h

```diff
@@ -176,18 +176,20 @@ typedef struct BoxPokemon {
 
 /**
  * @brief Party Pokemon data structure
+ *  This is used to store stats for a pokemon while it is in the players party.
+ *  Rather than recalculating stats after each battle, they're stored here.
  */
 typedef struct PartyPokemon {
-    u32 unk_00;
-    u8 level;  //!< The pokemons level
-    u8 unk_05;
-    u16 unk_06;
-    u16 unk_08;
-    u16 unk_0A;
-    u16 unk_0C;
-    u16 unk_0E;
-    u16 unk_10;
-    u16 unk_12;
+    u32 status;
+    u8 level;
+    u8 mail;
+    u16 hp;
+    u16 maxHP;
+    u16 attack;
+    u16 defense;
+    u16 speed;
+    u16 spAtk;
+    u16 spDef;
```

Comment:
In general, maintain the Doxygen comments on struct fields (and add them, if possible).

Reply:
oh, I'm sorry I'm fairly new to this and thought that was an excited comment, I'll try to add some. What does it do?

Reply:
re-added there

Reply:
The Doxygen comments are for whenever we want to generate documentation, similar to Javadoc or Sphinx: https://www.doxygen.nl/